goshopify.go

```
@@ -234,6 +241,11 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 	}
 
 	if callLimit, ok := resp.Header[callLimitHeader]; ok {
+		// Extract the current bucket capacity from the call limit response header
+		bucketCapacity, err = strconv.Atoi(strings.Split(callLimit[0], "/")[0])
+		if err != nil {
+			return err
```

Can we ignore this error and log warn report only. use . a default value if erroring

---

yes